features/foutafhandeling.feature

```diff
@@ -0,0 +1,542 @@
+
+        Voorbeelden:
+        | path                   | identificatie    | resource naam       |
+        | /adresseerbareobjecten | 1599010000048400 | adresseerbaarObject |
+        | /panden                | 1234567890123456 | pand                |
+
+Rule: wanneer gevraagd wordt om een contenttype anders dan json wordt een 406 Not Acceptable gegeven
+
+    Scenario: niet ondersteund contenttype
+        Als '/adressen/0599200000193766' request wordt gedaan met header 'Accept: application/xml'
+        Dan bevat de response ten minste de volgende velden
+        | naam   | waarde                                                                                                                    |
+        | title  | Gevraagde contenttype wordt niet ondersteund.                                                                             |
+        | status | 406                                                                                                                       |
+        | code   | notAcceptable                                                                                                             |
+        En bevat de response geen veld "invalidParams"
+        En is de http status code van het antwoord 406
+        En bevat de response de header "Content-Type" met waarde "application/problem+json"
+
+    Scenario: contenttype json
+        Als '/adressen/0599200000193766' request wordt gedaan met header 'Accept: application/json'
+        Dan is de http status code van het antwoord 200
+
+    Scenario: contenttype hal+json
+        Als '/adressen/0599200000193766' request wordt gedaan met header 'Accept: application/hal+json'
+        Dan is de http status code van het antwoord 200
+
+Rule: wanneer gevraagd wordt om geometrie in een crs anders dan RD (epsg:28992) wordt een 406 Not Acceptable gegeven
+
+    Scenario: accept-crs WGS84 geeft een foutmelding
+        Als '/adresseerbareobjecten/0599010000165822' request wordt gedaan met header 'Accept-Crs: epsg:4326'
+        Dan bevat de response ten minste de volgende velden
+        | naam   | waarde                                                         |
+        | title  | Gevraagde coördinatenstelsel EPSG:4326 wordt niet ondersteund. |
+        | status | 406                                                            |
+        | code   | crsNotAcceptable                                               |
+        En bevat de response geen veld "invalidParams"
+        En is de http status code van het antwoord 406
+        En bevat de response de header "Content-Type" met waarde "application/problem+json"
+
+    Scenario: accept-crs RD wordt ondersteund
+        Als '/adresseerbareobjecten/0599010000165822' request wordt gedaan met header 'Accept-Crs: epsg:28992'
+        Dan is de http status code van het antwoord 200
+
+Rule: wanneer er een technische fout optreed wordt een 500 Server Error gegeven
+
+    Scenario: interne server fout
```

Comment:
Hoe willen we de technische fout veroorzaken? Dit is anders niet te valideren

Reply:
dat is dus niet te valideren. als een gebruiker een server fout kan veroorzaken is dat een bug